website/content/en/highlights/2021-07-21-0-16-upgrade-guide.md

```diff
@@ -15,10 +15,7 @@ Vector's 0.16.0 release includes three **breaking changes**:
 1. [Component name field renamed to ID](#name-to-id)
 1. [Datadog Log sink encoding option removed](#encoding)
 1. [Renaming of `memory_use_bytes` internal metric](#memory_use_bytes)
-
-And one **deprecation**:
-
-1. [Vector source/sink version 2 released](#vector_source_sink)
```

Comment:
I moved this to its own file since we typically don't put deprecations in the upgrade guide.